cpp/tests/groupby/min_tests.cpp

```diff
@@ -429,5 +431,29 @@ TEST_F(groupby_min_struct_test, values_with_null_child)
   }
 }
 
+struct groupby_min_float_test : public cudf::test::BaseFixture {
+};
+
+TEST_F(groupby_min_float_test, values_with_infinity)
+{
+  using T          = double;
```

Comment:
Does the test also cover the `float` case?

Reply:
Good catch.  I change the test to cover both `float` and `double`.